### ISSUE TITLE
pytorch_model.bin stuck at 0%

### ISSUE THREAD

**Issue by khooinguyeen** [myshell-ai/MeloTTS#231]
Why when I run melo-ui, the download of pytorch_model.bin stuck at 0% forever? I have already had another meloTTS downloaded in another virtual env and it run perfectly fine, why when I install it in another env, it does not work. Below is the terminal output.

% melo "Text to read" output.wav --language EN
/opt/anaconda3/envs/mira-stream-final/lib/python3.10/site-packages/huggingface_hub/file_download.py:795: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(
/opt/anaconda3/envs/mira-stream-final/lib/python3.10/site-packages/torch/nn/utils/weight_norm.py:143: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
/Users/khooinguyeen/Documents/Internship/Mira-local/submodules/MeloTTS/melo/download_utils.py:64: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  return torch.load(ckpt_path, map_location=device)
 > Text split to sentences.
Text to read
 > ===========================
  0%|                                                                                                                                                 | 0/1 [00:00<?, ?it/s]/opt/anaconda3/envs/mira-stream-final/lib/python3.10/site-packages/huggingface_hub/file_download.py:795: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(
/opt/anaconda3/envs/mira-stream-final/lib/python3.10/site-packages/huggingface_hub/file_download.py:795: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(
                                                                                                                                                                           Error while downloading from https://cdn-lfs.hf.co/bert-base-uncased/097417381d6c7230bd9e3557456d726de6e83245ec8b24f529f60198a67b203a?response-content-disposition=inline%3B+filename*%3DUTF-8%27%27pytorch_model.bin%3B+filename%3D%22pytorch_model.bin%22%3B&response-content-type=application%2Foctet-stream&Expires=1737208547&Policy=eyJTdGF0ZW1lbnQiOlt7IkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTczNzIwODU0N319LCJSZXNvdXJjZSI6Imh0dHBzOi8vY2RuLWxmcy5oZi5jby9iZXJ0LWJhc2UtdW5jYXNlZC8wOTc0MTczODFkNmM3MjMwYmQ5ZTM1NTc0NTZkNzI2ZGU2ZTgzMjQ1ZWM4YjI0ZjUyOWY2MDE5OGE2N2IyMDNhP3Jlc3BvbnNlLWNvbnRlbnQtZGlzcG9zaXRpb249KiZyZXNwb25zZS1jb250ZW50LXR5cGU9KiJ9XX0_&Signature=jEz9pVS9evOUG5oyliK0fsKxt2hrlOnXEC9wAi0LuWBgK-V0RcaNHYDj-1%7EJNs5vR1OeoNrrxZM3lNsgCj7K-E3CmwUfAxVUvi5H5CMtmUWidCDB5NKRO2WTZsvJzGp21a-xF3r02DAjD3BFuxokQkGeIdi-Og67R8bpjF4DjHXAwJ0uJOplWr5eax%7E4spV3tqgpzQdng9gQ3VdeskAI9315pzvUebOoU5q16QzXm0E0cFAX4xQPRla2AIbBMxRsED0v9c7Imp2UEe5mc7-hqBJIr5UTdxjhwGvuZRqVqZZwyyjl7uTggsSFGi2KqwnQMXbRufCYku3la-q13JgDcA__&Key-Pair-Id=K3RPWS32NSSJCE: HTTPSConnectionPool(host='cdn-lfs.hf.co', port=443): Read timed out.
Trying to resume download...

**Comment by khooinguyeen** [comment]
when I use the old env, meloTTS works just fine but when I clone that env to a new env, meloTTS cannot run in it, pytorch_model.bin stuck at 0% forever
